### PR TITLE
Shared schemas

### PR DESCRIPTION
**Pull request by Aryllen**
This is the first (and largest) batch of shared schemas.

With the exception of PEC's RNA seq template, I've moved all the other PEC templates that we decided on having the same keys for to a shared folder. The RNA seq template for PEC has some extra items related to single cell RNA seq. I didn't want to mess anything up for you all so left that one there. However, there is both an rnaSeq and scrnaSeq template in the shared folder that could be used by PEC.

I am thinking we should probably come up with a naming convention for PEC vs AD vs shared templates. This way we do not have namespace clashes for consortium-specific templates, such as the manifest, biospecimen, or rnaSeq (if PEC sticks with the original rnaSeq). At the moment, the templates in this batch have the same naming structure as the others in this repo. I'll create an issue where we can document ideas and the conclusion for how to name these.

I am requesting to merge this into the refactored-schemas branch. Is something holding up this branch from being the master/main branch since it will have the most up to date information?

### CODE REVIEW

File: shared/assay_rnaSeq_metadata_template.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        "libraryID": {
+            "$ref": "sage.annotations-ngs.libraryID"
+        },
+        "assay": {
+          "$ref": "sage.annotations-experimentalData.assay"
+        },
+        "platform": {
+            "$ref": "sage.annotations-experimentalData.platform"
+        },
+        "RIN": {
```
Comment (cw-dvr8) [inline shared/assay_rnaSeq_metadata_template.json:18]: What is the difference between "RIN" and "organRIN"?

**Review by cw-dvr8 — CHANGES_REQUESTED** [review]
I just added the "chromatinAmountUnits" key to the ChIPSeq assay metadata template this morning, so please make sure that you've grabbed that version.

File: shared/assay_rnaSeq_metadata_template.json (hunk 1 above)
Comment (Aryllen) [reply shared/assay_rnaSeq_metadata_template.json:18]: This is the definition I see for organRIN: "A general quality metric of the organ or tissue, not necessarily a direct RIN measure of the sample used in data generation."

This is the definition I see for RIN: "RNA integrity number"

I think one is specific to the exact sample being used, while the other is a general measure for the whole organ.

**Comment by Aryllen** [comment]
I merged your current version of the refractored branch into this one before pushing it up today.

**Comment by cw-dvr8** [comment]
Thanks for doing this!

I'm thinking that for a naming convention for AD and PEC specific templates, we could put "AD" or "PEC" in front of the schema type, e.g. "PECassay.ChIPSeq" or "ADindividual.human".  What do you think?

I haven't pushed the branch into master because not all of the terms have made it into synapseAnnotations yet, e.g. reprogrammedCellType and the PEC-specific terms.

**Comment by Aryllen** [comment]
I 100% got distracted and did not make the naming issue that I mentioned making, yet. 🤦🏻

Adding a specific naming convention to only the PEC and AD templates works for me. Can we separate them with a period for readability?

PEC.assay.ChIPseq and AD.individual.human?

**Comment by avanlinden** [comment]
I second the period for readability -- keeps it from being alphabet soup with the assays that are acronyms.

**Comment by Aryllen** [comment]
Added drosophila biospecimen template to the shared set since it is not specific to AD (even though AD is the only one with this type of data at the moment).

@cw-dvr8, anything else that needs updating/changing in this one or can we merge?

**Review by cw-dvr8 — APPROVED** [review]

**Pull request merged**